src/main/java/net/minestom/server/item/metadata/PotionMeta.java

```diff
@@ -41,15 +41,15 @@ public void write(@NotNull TagWritable writer, @NotNull CustomPotionEffect value
     }).list().defaultValue(List.of());
     private static final Tag<Color> CUSTOM_POTION_COLOR = Tag.Integer("CustomPotionColor").path("display").map(Color::new, Color::asRGB);
 
-    public PotionType getPotionType() {
+    public @NotNull PotionType getPotionType() {
         return getTag(POTION_TYPE);
     }
 
-    public List<CustomPotionEffect> getCustomPotionEffects() {
+    public @NotNull List<CustomPotionEffect> getCustomPotionEffects() {
         return getTag(CUSTOM_POTION_EFFECTS);
     }
 
-    public Color getColor() {
+    public @Nullable Color getColor() {
```

Comment:
Same. I think that we need parity with the client.

Reply:
This is also supposed to be nullable, and if it is null, the client will calculate the color of the potion item based on the potion effects. If we default it every potion will be the same color.